Seatsio/Sources/SeatingChartConfig.swift

```diff
@@ -69,6 +69,7 @@ public class SeatingChartConfig: SeatsioConfig, Encodable {
     public private(set) var lockActiveFloor: Bool?
     public private(set) var showFloorElevator: Bool?
     public private(set) final var unifiedObjectPropertiesInCallbacks = true
+    public private(set) var ticketBuyerId: String?
```

Comment:
I see we added most other properties to `CodingKeys` - is that not necessary here?

Reply:
of course it is 🤦‍♂️ 
thanks